src/backend/common/queries/mobile_client_query.py

```diff
@@ -0,0 +1,34 @@
+from backend.common.consts.client_type import ClientType
+from backend.common.futures import TypedFuture
+from backend.common.models.mobile_client import MobileClient
+from backend.common.queries.database_query import DatabaseQuery
+from google.cloud import ndb
+from typing import List
+
+
+class MobileClientListQuery(DatabaseQuery[List[MobileClient]]):
+    @ndb.tasklet
+    def _query_async(
+        self, users: List[str], client_types: List[ClientType] = list(ClientType)
+    ) -> TypedFuture[List[MobileClient]]:
+        if not users or not client_types:
+            return []
+
+        client_types = [c.value for c in client_types]
```

Comment:
Is this needed? It may need to be a separate variable?

Reply:
TIL apparently not 😲

Reply:
yeah, since `IntEnum` extends `int`, it should be implicitly convertable